### PR TITLE
news landing page

### PR DESCRIPTION
**Pull request by hafsaumar**

### CODE REVIEW

File: app/src/config_constants.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-const LOCAL_TEST_ENVIRONMENT = false;
+const LOCAL_TEST_ENVIRONMENT = true;
```
Comment (thanh-code-hub) [inline app/src/config_constants.js]: This should be false 😃

**Comment by thanh-code-hub** [comment]
@hafsaumar I added a small change to display content as paragraphs. But I think we should also show author and time for each news

File: app/src/containers/ListOfNews/index.js [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+
+
+    componentDidMount() {
+        this.getNewsListAPI(() => {
+            this.setState({ isLoadingPage: false });
+        });
+    }
+
+    getNewsTiles(news, classes) {
+        const newsImage = "https://www.predictiveindex.com/wp-content/uploads/2019/02/PI-News-Icon-2.png"
```
Comment (nnduc1994) [inline app/src/containers/ListOfNews/index.js:72]: careful with copyright in here, we could ask the customer to provide some img

**Review by nnduc1994 — APPROVED** [review]

File: app/src/containers/NewsDashboard/index.js [hunk 3]
```diff
@@ -40,11 +40,11 @@ class NewsDashboard extends React.Component {
 						</div>
 					},
 					cellStyle: {
-						width: "50%",
+						width: "25%",
```
Comment (nnduc1994) [inline app/src/containers/NewsDashboard/index.js:43]: @bvth  could break something here? since it changes your component

**Pull request merged**